Sites/index.html

```diff
@@ -0,0 +1,138 @@
+                        beautifully.</div>
+                </figcaption>
+
+
+                <h3> Hour 7 [ 4:37 ] </h3> <img src="img/Hour-7.jpg"
+                    alt="selfie of three girls holding a basket of fries and a hotdog">
+                <figcaption> As a snack, we stopped by a foodtruck to get cheese fries and a
+                    Chicago-style hotdog ! T'was delicious.</figcaption>
+
+
+                <h3> Hour 8 [ 5:22 ] </h3> <img src="img/Hour-8.jpg"
+                    alt="picture of a window with words written in it's dust">
+                <figcaption> We took the pinkline on the above ground train to a neighboorhood called Pilson for some
+                    thrift and vintage shoppping. In this photo, someone had wrote "old werld" in the dust on a
+                    store window. I thought it was funny, thus, a picture was born.
+                </figcaption>
+
+
+                <h3> Hour 9 [ 6:41 ] </h3> <img src="img/Hour-9.jpg" alt="picture of the corner of a library bookshelf">
+                <figcaption> Nearby, we stopped into a bookstore. There were techinically no phones
+                    allowed for some reason but I was feling rebellious so this is a snapshot I took of the
+                    mystery
+                    section ! </figcaption>
+
+
+                <h3>Hour 10 [ 7: 10 ] </h3> <img src="img/Hour-10.jpg" alt="girl smiling outside of a bookshop window"
+                    .>
+                <figcaption> Outside another bookstore ! We love to browse ! </figcaption>
+
+
+                <h3> Hour 11 [ 8:13 ] </h3> <img src="img/Hour-11.jpg" alt="two girls each holding a slice of pizza">
+                <figcaption> Late dinner at Dough Bro's
+                    pizza ! It was hard to find thin crust pizza in Chicago but finally we satisfied our
+                    East Coast cravings ! </figcaption>
+
+
+                <h3> Hour 12 [ 9:00 ] </h3>
+                <img src="img/Hour-12.jpg" alt="image of the James M. Netherlander Theatre stage">
+                <figcaption>At the James M. Nederlander Theatre, we saw "A Bronx Tale"
+                    the musical. Less well known but nevertheless a great soundtrack. </figcaption>
+
+
+                <h3> Hour 13 [ 10:44 ] </h3> <img src="img/Hour-13.jpg"
+                    alt="Image of the Chicago Riverwalk sign in front of two buildings">
+                <figcaption> Torwards the end of the night, the weather got a lot colder and the sky a
+                    lot foggier. This is a picture of the skyskrapers as we walked around at night.
+                </figcaption>
+
+
+                <h3> Hour 14 [ 11:03 ] </h3> <img src="img/Hour-14.jpg" alt="image of skyscrapers at night">
+                <figcaption> This is another image where I tried to capture the buildings fading into
+                    the fog. A gloomy end to a sunny day. </figcaption>
+            </div>
+
```

Comment:
Need to close the `.container` `div`:
```suggestion
  </div>
```